<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>哔哩哔哩视频下载器 Beta v0.1 作者：菠萝小西瓜  φ(゜▽゜*)♪</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/ico/favicon.ico</normaloff>:/ico/favicon.ico</iconset>
  </property>
  <widget class="QListView" name="Download_List">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>911</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,50);color:white;

font-family: 微软雅黑;

font-size:15px</string>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Setting_Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,30);color:white;

font-family: 微软雅黑;

font-size:15px</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Download_Manage_Button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,30);color:white;

font-family: 微软雅黑;

font-size:15px</string>
   </property>
   <property name="text">
    <string>下载管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Sgin_out_Button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,30);color:white;

font-family: 微软雅黑;

font-size:15px</string>
   </property>
   <property name="text">
    <string>退出B站账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>571</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/jpg/background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Login_Username">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,30);color:white;

font-family: 微软雅黑;

font-size:15px</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Sgin_in_Button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,30);color:white;

font-family: 微软雅黑;

font-size:15px</string>
   </property>
   <property name="text">
    <string>登录B站账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Main_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,30);color:white;

font-family: 微软雅黑;

font-size:15px</string>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>Main_Button</zorder>
  <zorder>Sgin_in_Button</zorder>
  <zorder>Download_Manage_Button</zorder>
  <zorder>Setting_Button</zorder>
  <zorder>Sgin_out_Button</zorder>
  <zorder>Login_Username</zorder>
  <zorder>Download_List</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
